### PR TITLE
Updating Development document from source with web ui installation steps

### PR DESCRIPTION
**Pull request by kgrvamsi**
Updated the Dev installation steps from source with st2 web ui component.

### CODE REVIEW

File: docs/source/development/sources.rst [hunk 1]
```diff
@@ -171,6 +171,37 @@ Additional Makefile targets
...
+Install |st2| Web UI
+~~~~~~~~~~~~~~~~~~~~~~
+
+Installing the st2 Web UI.
+
+.. code-block:: bash
+
+    sudo apt install npm
+    sudo npm install -g n
+    sudo n v10.15.3
```
Comment (armab) [inline docs/source/development/sources.rst:184]: Can it be less specific? All we need is node 10:

```suggestion
    sudo n v10
```

**Comment by armab** [comment]
cc @guzzijones and @bgaeddert, as you might have opinions on this considering st2web development.

File: docs/source/development/sources.rst (hunk 1 above)
Comment (kgrvamsi) [reply docs/source/development/sources.rst:184]: @armab i tried with v10.4.1 and some of the packages reported that to have node version more than v10.15.3 so specified this version specifically

Comment (kgrvamsi) [reply docs/source/development/sources.rst:184]:
```
TASK [st2_dev : Running Lerna Bootstrap] ***************************************
fatal: [default]: FAILED! => {"changed": true, "cmd": ["lerna", "bootstrap"], "delta": "0:00:01.089484", "end": "2020-05-25 12:17:09.447870", "msg": "non-zero return code", "rc": 1, "start": "2020-05-25 12:17:08.358386", "stderr": "lerna notice cli v3.22.0\nlerna info bootstrap root only\nerror st2web@2.4.3: The engine \"node\" is incompatible with this module. Expected version \"10.15.3\". Got \"10.20.1\"\nerror Found incompatible module.\nlerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'st2web'\nlerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'st2web'", "stderr_lines": ["lerna notice cli v3.22.0", "lerna info bootstrap root only", "error st2web@2.4.3: The engine \"node\" is incompatible with this module. Expected version \"10.15.3\". Got \"10.20.1\"", "error Found incompatible module.", "lerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'st2web'", "lerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'st2web'"], "stdout": "yarn install v1.22.4\n[1/5] Validating package.json...\ninfo Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.", "stdout_lines": ["yarn install v1.22.4", "[1/5] Validating package.json...", "info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command."]}
```

Comment (kgrvamsi) [reply docs/source/development/sources.rst:184]: Once i change this line to "v10.20.1" this worked
https://github.com/StackStorm/st2web/blob/3d28beed70634846267462ff32ef09793e5a6042/package.json#L18

Comment (armab) [reply docs/source/development/sources.rst:184]: Thanks for more context!

**Pull request merged**